coderd/notifications/manager.go

```diff
@@ -56,7 +59,13 @@ type Manager struct {
 //
 // helpers is a map of template helpers which are used to customize notification messages to use global settings like
 // access URL etc.
-func NewManager(cfg codersdk.NotificationsConfig, store Store, log slog.Logger) (*Manager, error) {
+func NewManager(cfg codersdk.NotificationsConfig, store Store, metrics *Metrics, log slog.Logger) (*Manager, error) {
+	// TODO(dannyk): add the ability to use multiple notification methods.
```

Comment:
Will this be fixed in this PR, a follow-up, or should we track this via issue?

Reply:
In a follow-up, but the work is not yet planned.
This will need to be implemented once we expand beyond system-only notifications.
I think it's ok to leave untracked for now.

Reply:
The reason I lifted this up is that `// TODO`s in code is usually where TODO's go to die, which is why I prefer them either tracked or not important enough to mention. Not a blocker to leave it in, just a pet-peeve.

Exhibit A:

```
rg 'TODO\(mafredri\)' | wc -l
      31
```